packages/playwright-core/src/utils/isomorphic/builtins.ts

```diff
@@ -41,7 +43,8 @@ export namespace Builtins {
   export type Date = OriginalDate;
 }
 
-export function ensureBuiltins(global: typeof globalThis): Builtins {
+export function builtins(global?: typeof globalThis): Builtins {
```

Comment:
It's hard to keep track of where the builtins are initialized the first time. Since we need it to happen before user's code is loaded into the page's context, let's add a comment somewhere describing how it is supposed to work (addInitScript? etc) or maybe add an explicit call that would initialize it.